### DISCUSSION TITLE
Spotfiy Change

### DISCUSSION THREAD

**Discussion by RedEdge967** [blueedgetechno/win11React#347] [Ideas]
Hey there,
What if we use open.spotify.com instead of spotify remake code and make it available to install from the store? I can open a PR to add open.spotify.app. is it okay?

**Comment by blueedgetechno** [comment]
First of all its open.spotify.com and second it doesnot allow to Iframe. Otherwise It would have been really stupid for me to recreate something that already exist.

**Reply by RedEdge967** [reply]
Okay blue
